modules/isd/include/ResidueProteinProximityRestraint.h

```diff
@@ -0,0 +1,83 @@
+/**
+ *  \file IMP/isd/ResidueProteinProximityRestraint.h
+ *  \brief Restraint a selection of particles (eg. a residue or 
+ *  segment) to be within  a certain distance of a second 
+ *  selection of particles (eg. a protein). 
+ *  Use to model data from mutagenesis experiments that disrupt
+ *  protein-protein interactions.
+ *
+ *  Copyright 2007-2022 IMP Inventors. All rights reserved.
+ *
+ */
+
+#ifndef IMPISD_RESIDUE_BINDING_RESTRAINT_H
```

Comment:
Header guard should generally match the name of the file, i.e. `IMPISD_RESIDUE_PROTEIN_PROXIMITY_RESTRAINT_H`.

Reply:
Done